docs/experience/url-groups.md

```diff
@@ -0,0 +1,39 @@
+title: URL Groups
+description: Guide for using URL Groups in Experience
+
+Most websites follow certain URL patterns for their pages. For example, your website might show user profiles at `/user/123` or `/user/312`. Therefore, the URL pattern is `/user/{userId}`. Mapping these patterns with URL Groups in Sematext Experience can give you a better overview of the performance data for each part of your website. The idea is that all user profiles will have more or less the same performance characteristics, so it makes sense to monitor the "User Profile" page as a whole instead of monitoring individual user profile pages.
+
+If we keep in mind the above example, here are the differences you will see if you use Url Groups.
+
+Before using URL Groups to group all user profile pages, you would see each user profile page as a separate entry in Top Pages table on the Overview screen. Clicking on one of the profile pages, eg. `/user/123` will open the Page Details screen with charts and tables that show performance metrics for all visitors that visited this particular user profile page. All entries in the Page Loads table will have the same url: `user/123`.
```

Comment:
Before using URL Groups to group all user profile pages, you would see each user profile page as a separate entry in Top Pages table on the Overview screen.
->
Before using URL Groups to group all user profile pages, you would see each user profile page as a separate entry in the Top Pages table on the Overview screen.